packages/client-python/src/rocketride/types/client.py

```
@@ -42,20 +42,20 @@
         'uri': 'wss://server.example.com',
         'onEvent': my_event_handler
     }
-    
+
     # Type hints for callbacks
     async def handle_event(event: DAPMessage) -> None:
         print(f"Event: {event['event']}")
 """
 
-from typing import Any, Callable, Awaitable, TypedDict, Literal, Optional
+from typing import Any, Callable, Awaitable, TypedDict, Literal, Optional, Union
```

Ruff (0.15.6)

[warning] Import from `collections.abc` instead: `Callable`, `Awaitable`

Import from `collections.abc`

(UP035)